### ISSUE TITLE
Cummulative average formula error

### ISSUE THREAD

**Issue by michaelvsj** [simon-larsson/keras-swa#10]
It appears that the formula used to calculate the cummulative average of the weights is wrong. It is missing self.swa_freq in the denominator.

It should be:
```
        def _average_weights(self, epoch):

            return [
                (swa_w * ((epoch - self.start_epoch) / self.swa_freq) + w)
                / ((epoch - self.start_epoch) / self.swa_freq + 1)
                for swa_w, w in zip(self.swa_weights, self.model.get_weights())
            ]
```

**Comment by simon-larsson** [comment]
You are correct, it seems to have been lost when the cyclic schedule was added. Will fix, thanks!

**Comment by michaelvsj** [comment]
Thanks a lot for this project! Tensorflow/keras implementation is still not working and yours is very clearly written.

**Comment by simon-larsson** [comment]
Thank you. I will try to get this fixed pretty soon. But until then you should probably use `lr_schedule='constant'` which should work despite this oversight. According to one of the authors of the paper there is little to no reason to use the cyclic schedule.

**Comment by simon-larsson** [comment]
Fixed in [version 0.1.7](https://pypi.org/project/keras-swa/0.1.7/). Thank you @michaelvsj
